package com.kodilla.exception.main;
import com.kodilla.exception.test.FirstChallenge;

public class ExceptionModuleRunner {

    public static void main(String[] args) {

        System.out.println(new FirstChallenge().divide(3,0));

    }
}
